Project.toml

```diff
@@ -50,7 +50,7 @@ FillArrays = "0.7, 0.8, 0.9, 0.10, 0.11, 0.12, 0.13"
 FunctionChains = "0.1"
 IntervalSets = "0.7"
 InverseFunctions = "0.1"
-KernelAbstractions = "0.8"
+KernelAbstractions = "0.8, 0.9"
```

Comment:
Does this still support KernelAbstractions v0.8 if all Devices are renamed to Backend? I would assume this not to work with v0.8 anymore..

Reply:
Yes, I tested it with KernelAbstractions v0.8.